File: Employee.java
import java.util.ArrayList;

public class Employee {
    private String name;
    private String lastName;
    ArrayList<BankAccount> accounts;
    private char type;

    public Employee(String name, String lastName) {
        this.name = name;
        this.lastName = lastName;
        this.accounts = new ArrayList<>();
    }

    public Employee(String name, String lastName, long accountNumber, char type) {
        this(name, lastName);
        if (type == 'A' || type == 'B' || type == 'C') {
            addAccount(accountNumber, type);
            System.out.println("Cuenta bancaria agregada con éxito.");
        } else {
            System.out.println("Tipo de cuenta inválido. La cuenta debe ser de tipo 'A', 'B' o 'C'.");
        }
    }

    public void addAccount(long accountNumber, char type) {
        if (type == 'A' || type == 'B' || type == 'C') {
            this.accounts.add(new BankAccount(accountNumber, type, name, lastName));
            System.out.println("Cuenta bancaria agregada con éxito.");
        } else {
            System.out.println("Tipo de cuenta inválido. La cuenta debe ser de tipo 'A', 'B' o 'C'.");
        }
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public ArrayList<BankAccount> getAccounts() {
        return accounts;
    }

}

File: employee.py
from bank_account import BankAccount


class Employee:
    __name = ""
    __last_name = ""
    accounts = []
    tipo = ""

    def __init__(self, name, last_name, account_number, tipo):
        self.__name = name
        self.__last_name = last_name
        self.add_account(account_number, tipo)

    def add_account(self, account_number, tipo):
        if tipo == 'A' or tipo == 'B' or tipo == 'C':
            self.accounts.append(BankAccount(account_number, tipo, self.__name, self.__last_name))
            print("Dado de alta con éxito.")

        else:
            print("Tipo de cuenta inválido.")

    def get_name(self):
        return self.__name

    def set_name(self, name):
        self.__name = name

    def get_last_name(self):
        return self.__last_name

    def set_last_name(self, last_name):
        self.__last_name = last_name

    def get_accounts(self):
        return self.accounts
